### PR TITLE
added text_input visibility option

### PR DESCRIPTION
**Pull request by casperstorm**
This adds `new_buffer.input_visibility` to control input visibility.
If you have multiple buffers it can be annoying that the text jumps within the buffer when input appears, and disappears.

### CODE REVIEW

**Comment by casperstorm** [comment]
Rebased on top of #127.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Unreleased
 
+Added:
+
+- Configuration option `new_buffer.input_visibility` to control input field visibility: always shown or following the focused buffer.
```
Comment (tarkah) [inline CHANGELOG.md:5]: I feel like we should rename this config option to just `buffer` at this point. Maybe with serde alias to `new_buffer` for backwards compatibility.

Now that we don't have the settings vs config confusion, I'm not sure `new_buffer` here makes sense as the name.

**Review by tarkah — APPROVED** [review]
LGTM! Just one comment about config field renaming, lmk if you agree

File: CHANGELOG.md (hunk 1 above)
Comment (casperstorm) [reply CHANGELOG.md:5]: Agree!

**Pull request merged**